content/api/suppression-list.apib

```diff
@@ -320,97 +294,104 @@ If the X-MSYS-SUBACCOUNT header is not provided, a search will be performed acro
         }
 
 
-### Delete a List Entry [DELETE]
+### Insert or Update a Suppression [PUT /suppression-list/{recipient_email}]
```

Comment:
Change the uri parameter from `recipient_email` to `recipient` so that it matches the suppression object name.